Tests/scripts/hook_validations/integration.py

```diff
@@ -118,6 +119,15 @@ def is_insecure_configured_correctly(self):
         if insecure_field_name:
             return self.is_valid_param(insecure_field_name, 'Trust any certificate (not secure)')
 
+    def is_valid_category(self):
+        """Check that the integration category is in the schema."""
+        category = self.current_integration.get('category', None)
+        if not category or category not in INTEGRATION_CATEGORIES:
+            self._is_valid = False
+            print_error("The category '{}' is not in the integration schemas".format(category))
```

Comment:
Sorry that i missed this one 
```suggestion
            print_error("The category '{}' is not in the integration schemas, the valid options are:\n{}".format(category, '\n'.join(INTEGRATION_CATEGORIES)))
```